extensions/_beacons/http_status.py

```diff
@@ -111,30 +111,32 @@ def beacon(config):
         if r.status_code >= 500:
             log.debug('Response from status endpoint was invalid: '
                       '%s', r.status_code)
-         for json_response_item in sites_config.get('json_response', []):
+        for json_response_item in sites_config.get('json_response', []):
             if ':' in json_response_item['path']:
                 service = json_response_item['path'].split(':')[0]
                 service_value = json_response_item['path'].split(':')[1]
             if service in r.json():
                 if json_response_item['comp'] in comparisons:
                     comp = comparisons[json_response_item['comp']]
-                    if not comp(json_response_item['value'],
-                                r.json()[service][service_value]):
+                    json_response_value = json_response_item['value']
+                    response_service_value = salt.utils.traverse_dict(r.json(), '{}:{}'.format(service, service_value))
```

Comment:
These names are a bit confusing. I would use something like "expected_value" and "actual_value" or "received_value" or something like that.